src/main/webapp/index.html

```diff
@@ -50,14 +50,15 @@
       </div>
       <div class="inline">
         <form id="input-form">
-          <input type="text" class="input-text" id="input-text" name="input-text" placeholder="Enter your postcard text.."> 
+          <input type="text" class="input-text" id="input-text" maxlength="100" name="input-text" onkeydown="return (event.keyCode!=13);" placeholder="Enter your postcard text.."> 
+          <span id="result_user">0/100</span> 
```

Comment:
The maximum length of a message should be 300. https://github.com/googleinterns/step230-2020/pull/60

Reply:
Also, try to use hyphen as word separator in ID. https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters